documentation/modules/ROOT/pages/introduction.adoc

```diff
@@ -104,4 +104,23 @@ image::browser-preview.png[]
 .Just your average web browser
 image::browser-preview-window.png[]
 +
-. Feel free to type in the address of your favorite website to show that it's just a typical (chrome-based) web browser
\ No newline at end of file
+. Feel free to type in the address of your favorite website to show that it's just a typical (chrome-based) web browser
+
+[#cockpit]
+== 6. RHEL Web Console (Cockpit)
+
+The RHEL web console is a Red Hat Enterprise Linux 8 web-based interface designed for managing and monitoring your local system, as well as Linux servers located in your network environment. In today's lab you may also like to examine containers with the console. In RHEL 8, cockpit comes included but you can install it with the following command.
+----
+yum install -y cockpit
+----
+
+To enable cockpit you simply enable it as a service and ensure the firewall rules are in place.
+----
+sudo systemctl enable --now cockpit.socket
+firewall-cmd --add-service=cockpit --permanent
+firewall-cmd --reload
+----
```

Comment:
Hey @bfarr-rh, I think this should already be enabled on the code server instance.  Was that not your experience?  If not, it's probably a bug in the ansible workshop setup script.